src/vcpkg/postbuildlint.cpp

```diff
@@ -1062,6 +1118,8 @@ namespace vcpkg::PostBuildLint
         error_count += check_no_files_in_dir(fs, package_dir);
         error_count += check_no_files_in_dir(fs, package_dir / "debug");
         error_count += check_pkgconfig_dir_only_in_lib_dir(fs, package_dir);
+        error_count += check_no_absolute_paths_in(
+            fs, package_dir, std::vector<Path>{package_dir.native(), paths.installed.native()});
```

Comment:
We should maybe also check for `CURRENT_BUILDTREES_DIR`. For example the file `vcpkg_installed/x64-osx/tools/apr/bin/apr-1-config` contains the following code:
```
APR_SOURCE_DIR="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/apr/src/apr-1-3a6fcafcf9.clean"
APR_BUILD_DIR="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/apr/x64-osx-rel"
```
Otherwise there is no good alternative value for these variables.